### ISSUE TITLE
Planted crop tiles disappeared.

### ISSUE THREAD

**Issue by Headjack** [CleverRaven/Cataclysm-DDA#28376]
**Describe the bug**  
I had planted several tiles of crops and many of them vanished in a very clear segment with no explanation. It was all in the same map tile, and not all the crops were affected. It's like a cube of ground just decide to say No, these are gone now.

**To Reproduce**  
What I did was clear several Rubble tiles in a damaged restaurant, farm them, close it in on the sides some, and then left to get 2x4s in the woods. When i came back there were several missing as described. Savescumming to a bit before didnt change anything. Dont know if reproduces

**Expected behavior**  
Not that.

**Screenshots**  
seen below. The pattern was:
. . . . . . . 
^^^^^
^^^^^
. . . . . . . 
^^^^^
^^^^^
 Almost all the way to the back of this map tile. You can see upturned earth still at the ends from where i ran out of seeds.

![image](https://user-images.githubusercontent.com/6004660/53314385-e8857b80-3872-11e9-8c36-ad33af4d83d4.png)

**Versions and configuration(please complete the following information):**
 - OS: [win10
 - Game Version: cataclysmdda-0.C-37457-g958e17f
 - Graphics version: terminal
 - Mods loaded: more survival tools, makeshift item mod

**Comment by Headjack** [comment]
> **Describe the bug**
> I had planted several tiles of crops and many of them vanished in a very clear segment with no explanation. It was all in the same map tile, and not all the crops were affected. It's like a cube of ground just decide to say No, these are gone now.
> 
> **To Reproduce**
> Steps to reproduce the behavior:
> What I did was clear several Rubble tiles in a damaged restaurant, farm them, close it in on the sides some, and then left to get 2x4s at the woods. When i came back there were several missing as described. Savescumming to a bit before didnt change anything. I dont know if that will reproduce it ¯\_(ツ)_/¯
> 
> **Expected behavior**
> Not that.
> 
> **Screenshots**
> seen below. The pattern was:
> . . . . . . .
> ^^^^^
> ^^^^^
> . . . . . . .
> ^^^^^
> ^^^^^
> Almost all the way to the back of this map tile. You can see upturned earth still at the ends from where i ran out of seeds.
> 
> ![image](https://user-images.githubusercontent.com/6004660/53314385-e8857b80-3872-11e9-8c36-ad33af4d83d4.png)
> 
> **Versions and configuration(please complete the following information):**
> 
> * OS: [win10
> * Game Version: cataclysmdda-0.C-37457-g958e17f
> * Graphics version: terminal
> * Mods loaded: more survival tools, makeshift item mod

**Comment by Headjack** [comment]
I planted some more there, and it happened again in the same places, except one square.

**Comment by int-ua** [comment]
I wonder if it was radioactive if it was generated damaged. Anyway, can anyone confirm this on a latest version?

**Comment by OliveChukar** [comment]
Tested in ver 0.E-9072-g1bb53ba-dirty with core content pack dark days ahead only. Wandered around the world until I found a crater. Planted several types of seed then walked so the crops were out of the reality bubble. When I walked back crops within the crater were gone but those outside it were still there. Checked radiation levels and the spots where crops dissapeared  were all over 20. Plants on radioactive ground dissapearing may be intended behavior and not a bug.
